src/app/components/list/grid/index.html

```diff
@@ -8,6 +8,9 @@
     <p>
       The list view grid component displays a grid for a SKY UX-themed list of data. It is designed for use within <a routerLink="../overview">a list component</a>. Internally, the <sky-code>sky-list-view-grid</sky-code> uses <a routerLink="../../grid">the <sky-code>sky-grid</sky-code> component</a>, which contains documentation for <sky-code>sky-grid-column</sky-code>.
     </p>
+    <p>
+      To use the list view grid component, you must also install <sky-code>SkyListModule</sky-code> as a dependency to your project. For more information about <sky-code>SkyListModule</sky-code>, see <a routerLink="/components/list/overview">List overview</a>.
```

Comment:
I'd delete "also" in the first sentence and "more" in the second sentence.

And same comment as above about eventually moving this to the Installation section.